src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -176,7 +162,22 @@ public static ResultLevel CreateResultLevel(ResultLevelVersionOne v1ResultLevel)
             }
         }
 
-        public static ResultLevelVersionOne CreateResultLevel(ResultLevel v2ResultLevel)
+        public static ResultLevelVersionOne CreateResultLevelVersionOne(RuleConfigurationDefaultLevel v2DefaultLevel)
```

Comment:
Just curious why you moved it down in the file. Alphabetizing methods?

Reply:
Yep

Reply:
Err, I like having the v2 version above the v1 version